apps/sms/views/conversation/js/conversation.js

```diff
@@ -2705,11 +2712,6 @@ var ConversationView = {
   },
 
   onHeaderActivation: function conv_onHeaderActivation() {
-    // Do nothing while in participants list view.
-    if (!Navigation.isCurrentPanel('thread')) {
-      return;
-    }
```

Comment:
The header has "pointer-events: none" when in the composer and we now have a separate header in the participants list view. So this check is useless.